webpack/scenes/Subscriptions/components/SubscriptionsToolbar/SubscriptionsToolbar.js

```diff
@@ -61,12 +61,12 @@ const SubscriptionsToolbar = ({
 
             {isManifestImported &&
               <a
-                href={SUBSCRIPTION_WATCH_URL}
+                href={SUBSCRIPTIONS_SERVICE_URL}
                 className="btn btn-default"
                 target="_blank"
                 rel="noreferrer"
               >
-                {__('Subscription Watch')}
+                {__('View Subscription Usage')}
```

Comment:
why not name it `Subscriptions service`?

Reply:
I'm just going with the wording suggested by the RH doc writer.